src/main/java/net/minecraftforge/client/event/RenderLivingEvent.java

```diff
@@ -28,35 +28,54 @@
 {
     private final EntityLivingBase entity;
     private final RenderLivingBase<T> renderer;
+    private final float partialRenderTick;
     private final double x;
     private final double y;
     private final double z;
 
+    /*
+        Since adding partialRenderTick causes binary incompatibility,
+        this overload adds a partialRenderTick that simply acts like
+        the game has no values it needs to interpolate.
```

Comment:
This type of information belongs in the PR explanation on github, not in a comment here.
The comment should just explain that anyone calling it should use the constructor that has `partialRenderTick` instead.